vendor/github.com/shirou/gopsutil/process/process_linux.go

```diff
@@ -176,10 +176,12 @@ func (p *Process) Gids() ([]int32, error) {
 
 // Terminal returns a terminal which is associated with the process.
 func (p *Process) Terminal() (string, error) {
-	terminal, _, _, _, _, err := p.fillFromStat()
+	t, _, _, _, _, err := p.fillFromStat()
+	termmap, err := getTerminalMap()
```

Comment:
Check the error return from fillFromStat before calling getTerminalMap

Reply:
This proposed change has been included in the upstream vendor PR.